modules/nf-core/merfin/hist/tests/main.nf.test

```diff
@@ -0,0 +1,642 @@
+// nf-core modules test merfin/hist
+nextflow_process {
+
+    name "Test Process MERFIN_HIST"
+    script "../main.nf"
+    process "MERFIN_HIST"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "merfin"
+    tag "merfin/hist"
+
+    test("sarscov2 - interleaved illumina reads") {
+
+        setup {
+
+            run("MERYL_COUNT") {
+                script "../../../meryl/count/main.nf"
+                config "./modules.config"
+                tag    "meryl/count"
+                process {
+                    """
+                    input[0] = [
+                        [ id:'test', single_end:true ], // meta map
+                        [file(params.test_data['sarscov2']['illumina']['test_interleaved_fastq_gz'], checkIfExists: true)]
```

Comment:
Tests should use file URLs instead of test_data now, please change

Reply:
Update: please check #5281 for details on the new format, unfortunately it's not in the docs yet

Reply:
Done.